Cannot connect to an active CDP4-COMET server

### Prerequisites

- [x] I have written a descriptive issue title
- [x] I have verified that I am running the latest version of the DEH-Capella
- [x] I have searched [open](https://github.com/RHEAGROUP/DEH-Capella/issues) and [closed](https://github.com/RHEAGROUP/DEH-Capella/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description
<!-- A description of the bug or feature -->
Cannot connect to an active CDP4-COMET server. When trying to connect the following error prompts:
![image](https://github.com/RHEAGROUP/DEH-Capella/assets/154615975/0bf8b94b-96f6-4ade-a12a-cbac9c1e5423)

From the log:
[INFO ] 2024-01-03 09:54:13.456 [ForkJoinPool.commonPool-worker-3] SessionImpl - Open request https://dndmmbse.cdp4.org
[DEBUG] 2024-01-03 09:54:13.458 [ForkJoinPool.commonPool-worker-3] CdpServicesDal - Creating HttpClient without proxy.
[DEBUG] 2024-01-03 09:54:13.461 [ForkJoinPool.commonPool-worker-3] CdpServicesDal - CDP4Services Open 4d0ea3ebcd97: https://dndmmbse.cdp4.org/SiteDirectory%3Fextent=deep&includeReferenceData=false
[INFO ] 2024-01-03 09:54:13.737 [ForkJoinPool.commonPool-worker-3] CdpServicesDal - CDP4 Services responded in 274 [ms] to Open 4d0ea3ebcd97
[ERROR] 2024-01-03 09:54:13.737 [ForkJoinPool.commonPool-worker-3] CdpServicesDal - The data-source replied with code 404: Not Found
[ERROR] 2024-01-03 09:54:13.737 [ForkJoinPool.commonPool-worker-3] SessionImpl - Open request for https://dndmmbse.cdp4.org unexpectedly finished with an exception cdp4dal.exceptions.DalReadException: The data-source replied with code 404: Not Found


I have checked that the credentials are ok. And I can access the server in CDP4-COMET app.

 DEH-Capella does connect with the public CDP4 server ( https://cdp4services-public.rheagroup.com)

@samatrhea could it be related to the dndmmbse server version (8.0.0 RC18)? Antoine Théate tested it with a local server 8.0.0 RC18 and do not work also.

### Steps to Reproduce
<!-- List of steps, sample code, failing test or link to a project that reproduces the behavior -->
- Open Capella
- Click on: DEHP/ Open/Close Hub Panel
- Connections

### System Configuration
Capella version: 6.1 
Additional information: 

<!-- Thanks for reporting the issue to DEH-Capella! -->

--- a/DEHCapellaAdapter/.classpath
+++ b/DEHCapellaAdapter/.classpath
@@ -3,11 +3,8 @@
 	<classpathentry exported="true" kind="lib" path="lib/log4j-api-2.11.2.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/log4j-core-2.11.2.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/log4j-slf4j-impl-2.11.2.jar"/>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-11">
-		<attributes>
-			<attribute name="module" value="true"/>
-		</attributes>
-	</classpathentry>
+	<classpathentry kind="lib" path="lib/DEHCommonJ-1.1.0.jar"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-11"/>
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
 	<classpathentry kind="src" path="src"/>
 	<classpathentry kind="src" output="bin.tests" path="tests">
@@ -17,7 +14,6 @@
 		</attributes>
 	</classpathentry>
 	<classpathentry kind="src" path="resources"/>
-	<classpathentry kind="lib" path="lib/DEHCommonJ-1.0.0.jar" sourcepath="lib/DEHCommonJ-1.0.0-sources.jar"/>
 	<classpathentry kind="lib" path="miglayout15-swing.jar" sourcepath="miglayout-src.zip"/>
 	<classpathentry kind="output" path="bin"/>
 </classpath>

--- a/DEHCapellaAdapter/META-INF/MANIFEST.MF
+++ b/DEHCapellaAdapter/META-INF/MANIFEST.MF
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: DEH-CapellaAdapter
 Bundle-SymbolicName: com.rheagroup.dehcapellaadapter;singleton:=true
-Bundle-Version: 1.5.2
+Bundle-Version: 1.5.4
 Bundle-Vendor: RHEAGROUP
 Require-Bundle: org.eclipse.ui,
  org.polarsys.capella.core.model.handler;bundle-version="5.1.0";visibility:=reexport,
@@ -18,7 +18,7 @@ Bundle-ClassPath: .,
  lib/log4j-api-2.11.2.jar,
  lib/log4j-core-2.11.2.jar,
  lib/log4j-slf4j-impl-2.11.2.jar,
- lib/DEHCommonJ-1.0.0.jar,
+ lib/DEHCommonJ-1.1.0.jar,
  miglayout15-swing.jar
 Export-Package: App,
  Commands,

--- a/DEHCapellaAdapter/build.properties
+++ b/DEHCapellaAdapter/build.properties
@@ -5,7 +5,7 @@ bin.includes = plugin.xml,\
                .,\
                icons/,\
                log4j2.xml,\
-               lib/DEHCommonJ-1.0.0.jar,\
+               lib/DEHCommonJ-1.1.0.jar,\
                lib/log4j-api-2.11.2.jar,\
                lib/log4j-core-2.11.2.jar,\
                lib/log4j-slf4j-impl-2.11.2.jar

--- a/DEHCapellaAdapterFeature/feature.xml
+++ b/DEHCapellaAdapterFeature/feature.xml
@@ -2,7 +2,7 @@
 <feature
       id="DEHCapellaAdapterFeature"
       label="DEHCapellaAdapter"
-      version="1.5.2.qualifier"
+      version="1.5.4"
       provider-name="Rhea Group SA">
 
    <description url="www.rheagroup.com">

